packages/shared/src/components/comments/CommentMarkdown.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { ReactElement } from 'react';
```

Comment:
In time we might have another place where we will utilize html/markdown contents, so I think it is better to make the name more generalized. Also that this is not comment-specific.

One example is at profile details, soon enough we might allow markdown in their bio or somewhere else.

Reply:
Ah very good point Lee, thanks for this :D